addons/diagnostic/XEH_preInit.sqf

```diff
@@ -8,6 +8,10 @@ ADDON = false;
 #include "XEH_PREP.hpp"
 #include "initSettings.inc.sqf"
 
+GVAR(watchControls) = [controlNull, controlNull, controlNull, controlNull];
+GVAR(watchStatements) = ["","", "", ""];
+addMissionEventHandler ["Draw2D", {call FUNC(watchUpdate)}];
```

Comment:
From wiki for Draw2d, "Will stop executing when UI loses focus". Does it stop if game gets unfocused with `-noPause`? If so I would go for EachFrame.

Reply:
Keeps running with noPause